ocs_ci/helpers/helpers.py

```diff
@@ -978,7 +978,7 @@ def verify_block_pool_exists(pool_name):
     logger.info(f"Verifying that block pool {pool_name} exists")
     ct_pod = pod.get_ceph_tools_pod()
     try:
-        for pools in TimeoutSampler(60, 3, ct_pod.exec_ceph_cmd, "ceph osd lspools"):
+        for pools in TimeoutSampler(600, 3, ct_pod.exec_ceph_cmd, "ceph osd lspools"):
```

Comment:
Does this timeout need to be 10x of original timeout?

Reply:
Addressed in latest commit.